astroquery/exoplanet_orbit_database/data/exoplanet_orbit_database_units.json

```diff
@@ -54,9 +54,9 @@
     "LAMBDA": "deg",
     "LAMBDALOWER": "deg",
     "LAMBDAUPPER": "deg",
-    "LOGG": "deg",
```

Comment:
I'm certain this json is directly coming from the server, and thus should not be edited, unless there has been changes server side and the new response is different.
@bmorris3 can confirm that he indeed got the file directly, using the webform/API

Reply:
This JSON is not coming from the server, I wrote it by hand and this is indeed a typo.

Reply:
You mean as in `"deg"` instead of `"dex"`? Setting it to the latter (assuming that makes it a `Unit("dex")`) would have the advantage of enabling conversion to dimensionless fractions or `u.mag`, like
`logg.to(u.dimensionless_unscaled, equivalencies=u.logarithmic())`
but that should probably be done consistently with `FE` as well.